packages/cli/src/index.ts

```diff
@@ -146,23 +147,75 @@ yargs
       commandWrapper(handleJoin)(argv);
     }
   )
-
   .command(
-    'push [api] [maybeDestination] [maybeBranchName]',
-    'Push an API description to the Redocly API registry.',
+    'push-status [pushId]',
+    'Push status.',
     (yargs) =>
       yargs
-        .usage('push [api]')
-        .positional('api', { type: 'string' })
-        .positional('maybeDestination', { type: 'string' })
-        .hide('maybeDestination')
-        .hide('maybeBranchName')
+        .positional('pushId', {
+          description: 'Push id.',
+          type: 'string',
+          required: true,
+        })
         .option({
           organization: {
             description: 'Name of the organization to push to.',
             type: 'string',
+            required: true,
             alias: 'o',
           },
+          project: {
+            description: 'Name of the project to push to.',
+            type: 'string',
+            required: true,
+            alias: 'p',
+          },
+          domain: { description: 'Specify a domain.', alias: 'd', type: 'string' },
+          wait: {
+            description: 'Wait for build to finish.',
+            type: 'boolean',
+          },
+          'max-execution-time': {
+            description: 'Maximum execution time in seconds.',
+            type: 'number',
+            default: 600,
+          },
```

Comment:
If the user passes `--wait` without a value, it will overwrite the default value. and I'm not sure it's a good idea to have two types for one option (`boolean` and `number`)

Reply:
Could it be an optional `number` instead?

Reply:
I think we wanna mix unrelated things 
How will it be used in case we make it optional `number` ? 
Passing `--wait` without argument will not be possible I think, so what we have to pass when we wanna wait until end of deployments?

Reply:
> we wanna wait until end of deployments?

Is there a reason why we don't want to use some default  timeout (a big one) in that case? Is it practical to allow operations without any time limit?

Reply:
I think that the deployment process will depend on the size of the portal, so it is worthwhile to provide the ability to change this value. The time limit protects us from endless waiting if something goes wrong, so It seems that it is always better to have a time limit

Reply:
>  better to have a time limit

So it looks like those options don't make too much sense one without the other.

Reply:
>So it looks like those options don't make too much sense one without the other.

This is a fact

Reply:
If you leave this as it is, please make `wait` required for `max-execution-time`.